src/components/Clock.jsx

```diff
@@ -17,7 +17,7 @@ const Clock = () => {
 		return () => clearInterval(interval);
 	}, []);
 
-	return <div className="mt-4 h-10 text-center">{time}</div>;
+	return <div className="mt-8 h-10 text-center">{time}</div>;
```

Comment:
Eigenlijk zou je een component zoals een clock geen styling moeten laten bevatten. Dan heb je het voordeel dat je het eender waar kan gebruiken en zelf kan kiezen hoe het eruit ziet. Als je de styling in de component steekt zal de clock er overal hetzelfde uitzien